runtime/vm/createramclass.cpp

```diff
@@ -2921,6 +2921,21 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 	J9Module* module = NULL;
 #if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
 	UDATA romFieldCount = romClass->romFieldCount;
+	/*             Structure of Flattened Class Cache              */
```

Comment:
Define a structure called:

```
typedef struct J9FlattenedClassCacheHeader {
    j9object_t defaultValue;
    UDATA empty;
    UDATA size;
}
```

When you need to write of read the defaultValue, just cast flattenedClassCache to `(J9FlattenedClassCacheHeader *)`.

This will make the code more clear